core/templates/pages/about-foundation-page/about-foundation-page.component.html

```diff
@@ -225,9 +219,10 @@ <h1>{{ 'I18N_ABOUT_FOUNDATION_PAGE_SECTION_SIX' | translate }}</h1>
     margin: 30px auto;
   }
 
-  .about-foundation-page button:hover,
-  .about-foundation-page button:focus {
-    background-color: #3d9991;
+  :host {
+    .about-foundation-page-button-area a {
```

Comment:
This still reaches into the button. Can it be passed as a CSS class instead through the element attributes?

Reply:
Done